### ISSUE TITLE
NVDA is not reading the paragraph contained inside the div if the previous value is blank.

### ISSUE THREAD

**Issue by Unnaarun** [nvaccess/nvda#13654]
#### Windows version: Windows 10

UI : Angular 12

### Expected behavior: Have to read any html field after reading previous blank value.

![accessibility](https://user-images.githubusercontent.com/5380551/165982416-bc9d8584-ac61-4a53-9f93-b1b428c1e186.png)

After reading the third line of address value as blank it does not read or recognize Town or City field.

**Comment by Unnaarun** [comment]
![accessibility](https://user-images.githubusercontent.com/5380551/165981309-c540b1bb-2b61-416a-a888-2a7583d427b4.png)

After reading the third line of address value as blank it does not read or recognize  Town or City field.

**Comment by seanbudd** [comment]
Hi @Unnaarun,

We are unable to process this issue as it stands.
Please fill out the issue template fully.

Many developers, contributors and community members here are vision impaired.
Screenshots are not accessible. You need to provide an alternative description, and capture any relevant information from images.

To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

**Comment by seanbudd** [comment]
Once this is done, please let us know and we can reopen this issue or you can open a new issue.
